docs/topics/native/native-improving-compilation-time.md

```diff
@@ -153,7 +153,39 @@ This feature is [Experimental](components-stability.md#stability-levels-explaine
 add the `kotlin.incremental.native=true` option to your `gradle.properties` file. If you face any problems,
 create an [issue in YouTrack](https://kotl.in/issue).
 
+### Customize LLVM optimization passes
+
+The default behavior for an `-opt` compilation is to run the same LLVM optimization passes that clang would for C++.
+While this does provide good runtime performance, this may take too long to compile large apps. The LLVM optimizations
+are split into two parts; the module passes and the link-time passes. For a typical compilation, the two pipelines are
+run back to back, and the only real distinction is in the default values.
+
+To find out if one of the LLVM passes is taking too long, add the `-Xprofile-phases` option to your build, and examine
+the log output. There will be a separate section for each of the two LLVM pipelines. There is no direct way to disable a
```

Comment:
> add the `-Xprofile-phases` option to your build, and examine the log output.

add the `-Xprofile-phases` option to your build, run a single `linkRelease*` task and examine the log output.

Maybe also mention that `-i` Gradle option is required (if it is required).